src/Kmd.Logic.Audit.Client.SerilogLargeAuditEvents/SerilogLargeAuditEventClient.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using Serilog.Core;
+
+namespace Kmd.Logic.Audit.Client.SerilogLargeAuditEvents
+{
+    public class SerilogLargeAuditEventClient : IAudit, IDisposable
+    {
+        private readonly Logger customSinkLogger;
+        private readonly bool disposeLogger;
+        private readonly SerilogCustomSinkLoggerAudit auditToCustomSink;
+
+        public SerilogLargeAuditEventClient(SerilogLargeAuditEventClientConfiguration config)
```

Comment:
You have to provide `ctor` allowing client to provide own `blob client`
To reuse extension that you implement:
```
AzureBlobOrEventHub(
              this LoggerSinkConfiguration loggerConfiguration,
              string eventHubConnectionString,
              string eventHubName,
              int eventSizeLimitInBytes,
              BlobServiceClient blobServiceClient,
              string storageContainerName = null,
              ITextFormatter formatter = null)
```

Reply:
Added one more ctor to accept blob service client.